pmultiqc/modules/quantms/histogram.py

```diff
@@ -1,119 +1,104 @@
-from collections import OrderedDict, defaultdict
+from collections import OrderedDict
 
 class Histogram:
 
-    def classify(self, plot_cat, with_remains = False, data_cats = None):
-        # with_remains -> Whether there are any remaining values outside the statistical range.
-        # data_cats -> Make statistics and display according to different data categories.
-        # plot_cat = 'frequency' -> Count the number of occurrences of different values.
-        # plot_cat = 'range' -> Count the frequency with which a value occurs over several consecutive ranges.
-        # plot_cat = 'record' -> Assigns values in order according to the different ranges provided.
-
+    def __init__(self, description, plot_category, breaks = None):
+        # Initialise some parameters
+        self.description = description
         self.data = dict()
-        self.data_cats = data_cats
-        if plot_cat == 'frequency':
-            if with_remains:
-                self.plot_cat = 2
-            else:
-                self.plot_cat = 1
-                self.frequency = dict()
-        elif plot_cat == 'range':
-            self.plot_cat = 3
-        elif plot_cat == 'record':
-            self.plot_cat = 4
+        self.breaks = breaks
```

Comment:
Can you document in docstrings, what each of the members do?

Reply:
OK, I will do it.